index.rst

```diff
@@ -48,9 +48,118 @@
 
    **This technote is not yet published.**
 
-   Some of the provisioning steps in the current design require privileged operations; for those, we should move those functions to privileged initContainers so the main container runs only as the target user.  This proposal outlines how we can do that.
+Some of the provisioning steps in the current design require privileged
+operations; for those, we should move those functions to privileged
+initContainers so the main container runs only as the target user.  This
+proposal outlines how we can do that.
 
 .. Add content here.
+
+The Problem
+===========
+
+We most definitely do not want to maintain a separate container build
+per user.  Because of the way we are currently handling file permissions
+and collaboration within the RSP, we create a Unix user with UIDs and
+GIDs mapped from our authentication provider, and then we provision a
+home directory for that user if it does not exist.
+
+Currently, we use the ``provisionator`` (``769:769``) account to perform
+those actions, which are privileged.  In order to do that, the container
```

Comment:
What does "privileged" mean exactly? Does the provisioning just have to run as a specific UID (and if so, which one? if 0, does that mean that we don't have root squash turned on?)

Reply:
Provisionator runs as 769:769 and has specific sudoers rules controlling what it's allowed to be root to do.  We in fact are running no_root_squash everywhere but NCSA.